daemon/src/engine/client/cl_main.cpp

```diff
@@ -50,6 +50,8 @@ cvar_t *cl_wavefilerecord;
 #include "framework/Crypto.h"
 #include "framework/Network.h"
 
+#include "../common/Profiler.h"
```

Comment:
Do you think the profiles should be in Common.h? It doesn't seem something that must be visible everywhere, from what I understand it's going to be used only on a selected number of functions.